corehq/apps/commtrack/models.py

```diff
@@ -292,6 +292,9 @@ def sync_supply_point(location, is_deletion=False):
         updated_supply_point = _reopen_or_create_supply_point(location)
         location.supply_point_id = updated_supply_point.case_id
 
+    if update_fields:
```

Comment:
I think this and other similar conditions in this PR should check if `is not None` since I think `update_fields` could be an empty list, and this condition should always be triggered if it's a list.

Reply:
Good catch. Resolved in 7d63f0f856fc8bb92521e3efa0d7259458b06bd7